conn.go

```diff
@@ -635,18 +658,6 @@ func getTransport(address string) (transport, error) {
 	return nil, err
 }
 
-// dereferenceAll returns a slice that, assuming that vs is a slice of pointers
```

Comment:
This should really be removed in a separate commit as it is unrelated to adding the interceptors.

Reply:
This function is not used and it's not exported, extracted the change to a separate commit